test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1116,9 +1116,22 @@ Describe "WSMan Config Provider tab complete tests" -Tags Feature,RequireAdminOn
         $res = TabExpansion2 -inputScript $path -cursorColumn $path.Length
         $listener = Get-ChildItem WSMan:\localhost\Listener
         $res.CompletionMatches.Count | Should -Be $listener.Count
+
+        # Listeners will have no duplicate names.  Don't rely on order.
+        $match = $true
```

Comment:
Maybe this can be used instead:
```PowerShell
Describe "be-in test" { 
  It "order test" { 
    @(2,1) | Should -BeIn @(1,2,3,'a','b','c') 
  }
}
```